modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogsEditEntryDisplayContext.java

```diff
@@ -129,14 +131,23 @@ public String getDescription() {
 	}
 
 	public String getEditEntryURL() {
+		if (_portletResource.isEmpty()) {
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)_httpServletRequest.getAttribute(
+					WebKeys.THEME_DISPLAY);
+
+			_portletResource = themeDisplay.getPortletDisplay(
+			).getPortletResource();
```

Comment:
Never use method chaining in this way. Use an intermediate variable for the portlet display.